python/MooseDocs/extensions/tagging.py

```diff
@@ -102,15 +102,36 @@ def postExecute(self):
                     if (entry == 'path'):
                         path_value = tag_dict_str.split("path: ")[1].split(',')[0].replace("'", "")
 
+                        # splitting the path is dangerous, even at content/, as more than one tagged page could have
+                        # the same name.
                         if 'content/' in path_value:
-                            # splitting at content is dangerous, same named pages within two different modules/xxx/doc/content
-                            # could be mixed together
-                            path_value = path_value.split('content/')[1]
-
-                        # Use a single / to indicate that it's a local link
-                        link_value = '/' + path_value.replace('.md', '.html')
+                            path_value_cut = path_value.split('content/')[1]
+                        else:
+                            path_value_cut = path_value.split('/')[-1]
+
+                        # Find the relative path from the filter page (assumed at filter/index.html)
+                        # Check that there is no ambiguity
+                        target_page = self.translator.findPages(path_value_cut)
+                        filter_page = self.translator.findPages("filter/index.html")
+                        if len(target_page) != 1:
```

Comment:
why is this a warning?

Reply:
because it will inevitably happen. Cutting at content/ can yield conflicts with same-named pages

Reply:
the page will still be indexed though, there just wont be a link

Reply:
made them errors. People will have to de-conflict their pages if they want to use our shiny new extension

Reply:
I plan on finishing up the explicit fixes in my other PR for moosedocs now that the FY is done. I just verified that it'll work here too.

Reply:
I think warning to error is a good change for this particular check - it does keep it consistent with similar errors in when content is initially retrieved. Thanks @loganharbour